src/pages/our_ambassadors.astro

```diff
@@ -342,13 +326,12 @@ import AmbassadorCard from "@components/AmbassadorCard.astro";
         <a href="https://nf-co.re/governance#core-team" target="_blank">core team</a>.
       </AmbassadorCard>
 
-      <AmbassadorCard name="Janan Gawra" img="Janan.jpeg" country="es" github="JananGawra" linkedin="janangawra">
-        Dr. Janan Gawra is a biologist currently working as a postdoctoral researcher at the <a
+      <AmbassadorCard name="Janan Gawra" img="Janan.jpeg" country="es" github="JGawra" linkedin="janangawra">
+        Dr. Janan Gawra is as a postdoc researcher at the <a
```

Comment:
```suggestion
        Dr. Janan Gawra is a postdoctoral  researcher at the <a
```

Note the small typo; but I also suggest using "postdoctoral" instead of "postdoc" (unless it's a space thing?)

Reply:
Thanks for spotting the typo! Initially it was a space thing, but checking now, we can keep the postdoctoral. Removing the "as" probably made it possible 🤩